packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -0,0 +1,17 @@
+import { memoizeFunction } from '../../Utilities';
+import { mergeStyles, IStyle } from '../../Styling';
+import { IContextualMenuStyles } from './ContextualMenu.Props';
+
+export interface IContextualMenuClassNames {
+  title: string;
+}
+
+export const getClassNames = memoizeFunction((
```

Comment:
Could you merge the .styles file into classNames? I did that with toggle to simplify the pattern.